raiden/transfer/mediated_transfer/events.py

```diff
@@ -562,6 +564,25 @@ def from_dict(cls, data: Dict[str, Any]) -> 'EventUnlockFailed':
         return restored
 
 
+class EventNewBalanceProofReceived(Event):
+    """ Event for newly received balance proofs. Useful for notifying monitoring services. """
+
+    def __init__(self, balance_proof: BalanceProofSignedState) -> None:
```

Comment:
No need for `-> None`

Reply:
https://github.com/raiden-network/raiden/pull/3374#discussion_r253011733